ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-isis-new.cfg

```diff
@@ -0,0 +1,91 @@
+!RANCID-CONTENT-TYPE: arista
+!
+transceiver qsfp default-mode 4x10G
+!
+hostname router-isis-new
+!
+no aaa root
+no enable password
+!
+interface Ethernet1
+   description P2P_LINK_TO_EAPI-SPINE1_Ethernet1
+   mtu 1500
+   no switchport
+   ip address 172.31.255.1/31
+   isis enable EVPN_UNDERLAY
+   isis metric 50
+   isis network point-to-point
+!
+interface Ethernet2
+   description P2P_LINK_TO_EAPI-SPINE2_Ethernet1
+   mtu 1500
+   no switchport
+   ip address 172.31.255.3/31
+   isis enable EVPN_UNDERLAY
+   isis metric 50
+   isis network point-to-point
+!
+interface Ethernet3
+   description MLAG_PEER_EAPI-LEAF1B_Ethernet3
+   channel-group 3 mode active
+!
+interface Loopback0
+   description EVPN_Overlay_Peering
+   ip address 192.168.255.3/32
+   isis enable EVPN_UNDERLAY
+   isis passive
+!
+interface Loopback1
+   description VTEP_VXLAN_Tunnel_Source
+   ip address 192.168.254.3/32
+   isis enable EVPN_UNDERLAY
+   isis passive
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.254.11/24
+!
+interface Vlan110
+   description PR01-DEMO
+   no shutdown
+   vrf TENANT_A_PROJECT01
+   ip address virtual 10.1.10.254/24
+!
+interface Vlan4093
+   description MLAG_PEER_L3_PEERING
+   ip address 10.255.251.0/31
+   isis enable EVPN_UNDERLAY
+   isis metric 50
+   isis network point-to-point
+!
+interface Vlan4094
+   description MLAG_PEER
+   mtu 1500
+   no autostate
+   ip address 10.255.252.0/31
+!
+router isis EVPN_UNDERLAY
+   net 49.0001.0001.0001.0001.00
+   is-type level-2
+   router-id ipv4 192.168.255.3
+   log-adjacency-changes
+   mpls ldp sync default
+   timers local-convergence-delay 15000 protected-prefixes
+   advertise passive-only
+   !
+   address-family ipv4 unicast
+      maximum-paths 4
+      fast-reroute ti-lfa mode link-protection level-2
+      fast-reroute ti-lfa srlg strict
+   !
+   address-family ipv6 unicast
+      maximum-paths 4
+      fast-reroute ti-lfa mode node-protection
+      fast-reroute ti-lfa srlg
+   !
+   segment-routing mpls
+      router-id 192.168.255.3
```

Comment:
it looks like EOS is deprecating this router-id knob. I think we should remove this knob again.

From my config test on 4.26.1F:
```
R3(config-router-isis-af)#   !
R3(config-router-isis-af)#   segment-routing mpls
R3(config-router-isis-sr-mpls)#      router-id 192.168.255.3
! Router ID configured under segment-routing mode will be overridden by the general or the ISIS router ID
R3(config-router-isis-sr-mpls)#      no shutdown
R3(config-router-isis-sr-mpls)#?
  adjacency-segment   Adjacency Segment allocation options
  prefix-segment      Configure a prefix segment
  proxy-node-segment  Configure a node segment on behalf of another node
  segment             Configuration for segments
  shutdown            Disable segment routing
  ----------------------------------------
  comment             Up to 240 characters, comment for this mode
  default             Set a command to its defaults
  exit                Leave ISIS segment-routing mpls configuration mode
  no                  Disable the command that follows
  show                Display details of switch operation
  !!                  Append to comment
```

Reply:
Got it, removed this knob again.